web/src/beta/lib/core/StoryPanel/Block/builtin/Video/index.tsx

```diff
@@ -1,16 +1,39 @@
-import type { CommonProps as BlockProps } from "../../types";
-import BlockWrapper from "../common/Wrapper";
+import { useMemo } from "react";
+import Player from "react-player";
+
+import type { CommonProps as BlockProps } from "@reearth//beta/lib/core/StoryPanel/Block/types";
+import BlockWrapper from "@reearth/beta/lib/core/StoryPanel/Block/builtin/common/Wrapper";
+import { getFieldValue } from "@reearth/beta/lib/core/StoryPanel/utils";
+import type { ValueTypes } from "@reearth/beta/utils/value";
 
 const VideoBlock: React.FC<BlockProps> = ({ block, isSelected, ...props }) => {
+  const src = useMemo(
+    () => getFieldValue(block?.property?.items ?? [], "src") as ValueTypes["string"],
+    [block?.property?.items],
+  );
+  const title = useMemo(() => block?.property?.items?.[1]?.title, [block?.property?.items]);
+
   return (
     <BlockWrapper
-      title={block?.title}
+      title={title}
       icon={block?.extensionId}
       isSelected={isSelected}
       propertyId={block?.property?.id}
       propertyItems={block?.property?.items}
-      {...props}
-    />
+      {...props}>
+      {src && (
+        <Player
+          url={src}
+          width="100%"
+          height="255px"
```

Comment:
Do we have to set the height? I feel like we will want the height to be the height of the video, similar to the image. Not all vids will be 255px height.

Reply:
agreed , done